src/bigquery.ts

```diff
@@ -1,48 +1,26 @@
 import { BigQuery } from '@google-cloud/bigquery'
 import { parseJSON } from 'date-fns'
+import { SearchParams, Platform } from './types'
 
-import { SearchParams } from './types'
-
-import { sleep } from './utils/promise'
 import logger from './logger'
 import { ENVIRONMENT } from './config'
 
-const projectId = ENVIRONMENT === 'prod' ? 'entur-prod' : ENVIRONMENT
+const projectId = `entur-${ENVIRONMENT}`
 const bigQuery = new BigQuery({ projectId })
 
-const MAX_RETRIES = 5
-
-async function queryWithRetries(query: string, retriesDone = 0): Promise<void> {
```

Comment:
Fjernet dette siden retries ikke ble brukt. Ser forøvrig at bigQuery har en autoRetry og maxRetries funksjon bygd inn som man kan stille inn på om ønsket.

Reply:
Men den defaultet vel til 0 retries, siden det ikke ble spesifisert i kallet? Slik jeg leser det i hvert fall. Kan godt teste litt og legge til autoRetry: true, i ny PR.

Reply:
Det er "omvendt", den teller frå 0 til 5 (MAX_RETRIES). Kanskje ikkje det mest naturlige. :)